novawallet/Modules/NetworkDetails/View/NetworkDetailsNodeView.swift

```diff
@@ -0,0 +1,169 @@
+import SoraUI
+
+class NetworkDetailsNodeTableViewCell: PlainBaseTableViewCell<NetworkDetailsNodeView>, TableViewCellPositioning {
+    let separatorView: BorderedContainerView = .create { view in
+        view.strokeWidth = 0.5
+        view.strokeColor = R.color.colorDivider()!
+        view.borderType = .bottom
+    }
+
+    override func setupStyle() {
+        super.setupStyle()
+
+        backgroundColor = .clear
+        selectionStyle = .none
+    }
+
+    override func setupLayout() {
+        super.setupLayout()
+
+        contentDisplayView.roundedContainerView.backgroundView.addSubview(separatorView)
+        separatorView.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(16)
+            make.bottom.top.equalToSuperview().inset(separatorView.strokeWidth)
+        }
+    }
+
+    override func setHighlighted(_ highlighted: Bool, animated: Bool) {
+        super.setHighlighted(highlighted, animated: animated)
+
+        contentDisplayView.roundedContainerView.backgroundView.fillColor = highlighted
```

Comment:
I think we can set highlightedFillColor for the RoundedView and set its isHighlighted flag